best-practices/README.md

```diff
@@ -173,12 +173,13 @@ Email
 
 * Use [SendGrid] or [Amazon SES] to deliver email in staging and production
   environments.
-* Use a tool like [MailView] to look at each created or updated mailer view
-  before merging.
+* Use a tool like [ActionMailer Preview] to look at each created or updated mailer view
+  before merging. Use [MailView] gem unless using Rails version 4.1.0 or later.
 
 [Amazon SES]: http://robots.thoughtbot.com/post/3105121049/delivering-email-with-amazon-ses-in-a-rails-3-app
 [SendGrid]: https://devcenter.heroku.com/articles/sendgrid
 [MailView]: https://github.com/37signals/mail_view
```

Comment:
I can remove the reference and link if you like? But Rails 4.1 isn't *really* old so previous version will be around and MailView is pretty useful tool still - don't want them to feel left out :innocent: